cassandra-analytics-core/src/main/java/org/apache/cassandra/clients/Sidecar.java

```diff
@@ -129,6 +135,9 @@ public static SidecarClient from(SidecarInstancesProvider sidecarInstancesProvid
                                             .ssl(conf.hasKeystoreAndKeystorePassword())
                                             .build();
 
+        StartupValidator.instance().register(new KeystoreValidation(secretsProvider));
+        StartupValidator.instance().register(new TruststoreValidation(secretsProvider));
+
         SidecarConfig sidecarConfig = new SidecarConfig.Builder()
```

Comment:
Unrelated to this PR, but there was a refactoring of configuration, so this class (SidecarConfig) has been refactored to SidecarClientConfig. You'll need to do something like this to make it build:

```
SidecarClientConfig sidecarConfig = SidecarClientConfigImpl.builder()
```

Reply:
you'll need this to make it compile

Reply:
I am confused by this comment, seems to still exist/compile for me.

Reply:
Done.